src/StringUtils.cpp

```diff
@@ -140,6 +140,54 @@ AStringVector StringSplit(const AString & str, const AString & delim)
 
 
 
+AStringVector StringSplitWithQuotes(const AString & str, const AString & delim)
+{
+	AStringVector results;
+
+	size_t cutAt = 0;
+	size_t Prev = 0;
+	size_t cutAtQuote = 0;
+
+	while ((cutAt = str.find_first_of(delim, Prev)) != str.npos)
+	{
+		AString current = str.substr(Prev, cutAt - Prev);
+		if ((current.at(0) == '"') || (current.at(0) == '\''))
```

Comment:
Is std::string::at preferred, as opposed to unchecked [] operators? I understand that since it throws exceptions and no exception handlers exist around here, it provides no benefits whilst negatively impacting speed.